Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.2-alpine3.19
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Ensure that the `COPY . .` command is intended to copy all files from the current directory on the host to the working directory in the container. This can lead to large image sizes if not managed carefully.